tests/testgroup40.py

```diff
@@ -1018,7 +1019,7 @@ def runTest(self):
         self.assertEqual(rc, 0, "Failed to delete all flows")
 
         logging.info("Inserting flow entry with hard_timeout set and send_flow_removed_message flag set")
-	   
+        
         # Insert a flow with hard_timeout = 1 but no Send_Flow_Rem flag set
         pkt = simple_tcp_packet()
```

Comment:
`pkt` is declared in line number `1044`.Can get rid of this line.Everything else looks good to me.
@jonstout any comments?